Server/Python/src/dbs/business/DBSBlock.py

```diff
@@ -8,6 +8,16 @@
 from dbs.utils.dbsExceptionHandler import dbsExceptionHandler
 import re
 
+
+def convertByteStr(unicodeStr):
+    """
+    Utilitarian function which converts an unicode string to
+    an 8-bit string.
+    """
+    if isinstance(unicodeStr, basestring):
+        return unicodeStr.encode("utf-8")
```

Comment:
why do we need to convert to `utf-8` and not to `ascii`? Do we store dataset/block/lfn as utf8?

Reply:
I guess it's a question to Yuyi. utf-8 supports ascii, so we should be fine as well (not sure about database performance though).